astropy/__init__.py

```diff
@@ -296,6 +296,19 @@ def _rebuild_extensions():
                       'with error code {0}: try rerunning this command '
                       'manually to check what the error was.'.format(
                           sp.returncode))
+# Set the bibtex entry to the article referenced in CITATION
+def _get_bibtex():
+    import os
+    import re
```

Comment:
These imports can just go at the top of the module.  I'm certain `os` at least has got to be already there:)
We also need to update the package setup to include the `CITATION` file in installs (and then get rid of the `if os.path.exists` clause).